### DISCUSSION TITLE
Which do you install? resources or mcuboot or both? Does it matter which you install first?

### DISCUSSION THREAD

**Discussion by djpeter84** [InfiniTimeOrg/InfiniTime#1953] [General]
infinitime-resources-1.14.0.zip
pinetime-mcuboot-app-dfu-1.14.0.zip

**Comment by FintasticMan** [comment]
You should install both. The DFU file is the firmware, the resource file contains fonts and images. All companion apps should be able to update the firmware, but only ITD, Amazfish and Watchmate support the resource file for now. You can also use [InfiniTimeExplorer](https://infinitimeexplorer.netlify.app/), but I think you then have to move the files manually. It doesn't matter which order you install them in.
